app/src/main/java/com/example/concordia_campus_guide/Models/Building.java

```diff
@@ -0,0 +1,86 @@
+package com.example.concordia_campus_guide.Models;
+
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.List;
+
+public class Building extends Place {
```

Comment:
just a heads up, this will need to be modified as Thomas and my changes added more functionality and attributes to this class

Reply:
I understand, I'll merge on top of the other changes before finalizing this PR.